src/js/pensions/config/form.js

```diff
@@ -136,9 +141,46 @@ const formConfig = {
               combatSince911
             }
           }
+        },
+        reserveAndNationalGuard: {
+          path: 'military/reserve-national-guard',
+          title: 'Reserve and National Guard',
+          uiSchema: {
+            'ui:title': 'Reserve and National Guard',
+            nationalGuardActivation: {
+              'ui:title': 'Are you currently on federal active duty in the National Guard?',
+              'ui:widget': 'yesNo'
+            },
+            nationalGuard: {
+              'ui:options': {
+                expandUnder: 'nationalGuardActivation',
+              },
+              name: {
+                'ui:title': 'Name of Reserve/NG unit',
+              },
+              address: address.uiSchema('Unit address'),
+              phone: phoneUI('Unit phone number'),
+              date: dateUI('Service Activation Date')
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['nationalGuardActivation'],
+            properties: {
+              nationalGuardActivation,
+              nationalGuard: {
```

Comment:
I think this would work, too:

`nationalGuard: _.set('properties.address', address.schema(fullSchemaPensions), nationalGuard)`

It looks like you only need to change the address, so doing it this way keeps us more in sync with the backend schema.